tests/test_frameSetUtils.py

```diff
@@ -73,6 +73,18 @@ def getCrpix(self, metadata):
             metadata.getScalar("CRPIX2") + metadata.getScalar("CRVAL2A") - 1,
         )
 
+    def testIgnoreNan(self):
+        """Test that NaN in a property list does not crash the WCS extraction.
+        """
+        metadata = self.makeMetadata()
+        metadata["ISNAN"] = float("NaN")
+
+        # This will issue a warning from C++ but that warning can not be
+        # redirected to python logging machinery.
+        # This code causes a SEGV without DM-21355 implemented
+        frameSet1 = readFitsWcs(metadata, strip=False)
+        self.assertEqual(type(frameSet1), ast.FrameSet)
+
```

Comment:
It makes sense that this test `SIGABRT`s on master, this was an untested edge case that could lead to an `SIGABRT` and it should now be tested (and fixed).